ui/packages/catalog-ui-search/src/main/webapp/js/model/List.js

```diff
@@ -79,7 +79,7 @@ module.exports = Backbone.AssociatedModel.extend({
     initialize: function() {
         this.set('query', new Query.Model({
             cql: generateCql(this.get('list.bookmarks')),
-            federation: 'enterprise'
```

Comment:
I don't know. I seem to recall that we made this change so that GSR could create lists, otherwise, they would not display. Do we currently expect metacards from non-local sources to be added to lists? @millerw8